services/port_manager/src/main/java/com/futurewei/alcor/portmanager/entity/SubnetPortIds.java

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright 2019 The Alcor Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+        you may not use this file except in compliance with the License.
+        You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+        Unless required by applicable law or agreed to in writing, software
+        distributed under the License is distributed on an "AS IS" BASIS,
+        WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+        See the License for the specific language governing permissions and
+        limitations under the License.
+*/
+package com.futurewei.alcor.portmanager.entity;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.Set;
+
+public class SubnetPortIds {
+    @JsonProperty("subnet_id")
+    private String subnetId;
+
+    @JsonProperty("port_ids")
+    private Set<String> portIds;
+
```

Comment:
@chenpiaoping Should include gateway port here and  distinguish it from a normal port? or add another field for gateway ports?

Reply:
Right now portIds only contains non-gateway port.

Reply:
ok